src/widgets/mainwindow.cpp

```diff
@@ -487,7 +487,7 @@ void MainWindow::setupToolBar()
                               | Qt::WindowCloseButtonHint | Qt::WindowFullscreenButtonHint;
 
         auto winFlags = windowFlags();
-        winFlags |= Qt::CustomizeWindowHint;
+        winFlags |= Qt::CustomizeWindowHint | Qt::FramelessWindowHint;
```

Comment:
Yes, adding this flag could eliminate the top blank line. But now we could not resize the main window by dragging on the border of it. Do you have any idea for this? Is it possible to enable tracking the mouse and resize the main window when the mouse is dragged alongside the border?